### ISSUE TITLE
Poor Performance

### ISSUE THREAD

**Issue by ParkerRS** [kiooeht/ModTheSpire#111]
I recently helped a friend install ModTheSpire after enjoying it and other mods myself. I walked him through it the exact way I did it, but with ModTheSpire and just one other mod installed, he gets insanely poor performance. I do not have issues with the same mods, and he has no issue when running through steam, modless. Does anyone else have this experience?

**Comment by WolfHeroEX** [comment]
Yes. On occasion, I've even had the game's audio start to stutter as if it was planning to induce a blue screen. Never had another game do that to me. Slay the spire unmodded through steam behaves perfectly for me.

Windows 10 64x Home edition. Could be a java issue?
